.github/workflows/upload-artifacts.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
           JRELEASER_GPG_PASSPHRASE: ${{ secrets.MAVEN_CENTRAL_GPG_PASSPHRASE }}
           JRELEASER_GPG_PUBLIC_KEY: ${{ secrets.MAVEN_CENTRAL_GPG_PUBLIC_KEY }}
           JRELEASER_GPG_SECRET_KEY: ${{ secrets.MAVEN_CENTRAL_GPG_SECRET_KEY }}
-        run: ./gradlew jreleaserDeploy
+        run: ./gradlew jreleaserDeploy -Dorg.gradle.jvmargs="-Xmx4g"
```

Comment:
Should we also increase it for snapshot releases? Or, is it a release-deploy specific issue?

Reply:
Good point! I only encountered the OOM error when publishing to Maven Central, but theoretically, the same issue could occur when publishing to the Maven Snapshot repository as well.
I’ll add the parameter for the snapshot release too. Thanks!

Reply:
Fixed in 7573e83a76544b329722d39756634c16390a5358. Thanks!

Reply:
Good to know that. Thank you for the fix!